src/java.base/share/classes/jdk/internal/util/Exceptions.java

```diff
@@ -0,0 +1,322 @@
+/*
+ * Copyright (c) 2018, 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
new file => copyright ?

Copyright (c) 2025,

Reply:
The file isn't actually new, though webrev doesn't seem to realise that on this occasion (sometimes it does recognise name changes). It was previously named SocketExceptions. I presume we keep the existing header in that case?